code/game/objects/structures/statues.dm

```diff
@@ -164,22 +166,27 @@
 
 /obj/structure/statue/gold/hos
 	name = "statue of the head of security"
+	desc = "A highly valuable statue of the Head of Security made from gold. Antagonists beware!"
 	icon_state = "hos"
```

Comment:
Antagonists is an ooc term

Reply:
Yes, however OOC terms are sometimes used to describe pbjects, such as in the mirror's description.